docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md

```diff
@@ -0,0 +1,129 @@
+# Single Cluster Open Source Observability - Istio Monitoring
+
+## Objective
+
+This pattern demonstrates how to use the _New EKS Cluster Open Source Observability Accelerator_ with Istio based workloads.
```

Comment:
Can you talk a little bit about the background what is istio, what is it for monitoring istio and what we are trying to accomplish. You can refer our blogs.